website/app/styles/doc-components/layout.scss

```diff
@@ -0,0 +1,26 @@
+.doc-layout {
+  display: flex;
+  flex-flow: wrap;
+  gap: var(--doc-layout-spacing);
+}
+
+// Direction:
+
+.doc-layout--direction-horizontal {
+  flex-direction: row; // this is the browser default
+}
```

Comment:
This style may not be needed since it's the browser default but would like to know opinions...

Reply:
Generally, I don't mind even if it's the default. In this way CSS and JS are decoupled.